### PR TITLE
EVG-1976: add cli commands for setting banner text

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: rest/client/rest.go [hunk 1]
```diff
@@ -126,6 +125,77 @@ func (c *communicatorImpl) GetHosts(ctx context.Context, f func([]*model.APIHost
...
+	}
+
+	return nil
+}
+
+func (c *communicatorImpl) GetServiceFlags(ctx context.Context) (*model.APIServiceFlags, error) {
+	info := requestInfo{
+		method:  get,
+		version: apiVersion2,
+		path:    "/admin/service_flags",
```
Comment (john-m-liu) [inline rest/client/rest.go]: you can't get this endpoint

Comment (tychoish) [reply rest/client/rest.go]: fixed

File: rest/client/rest.go [hunk 2]
```diff
@@ -126,6 +125,77 @@ func (c *communicatorImpl) GetHosts(ctx context.Context, f func([]*model.APIHost
 	return nil
 }
 
+func (c *communicatorImpl) SetBannerMessage(ctx context.Context, m string) error {
+	info := requestInfo{
+		method:  post,
+		version: apiVersion2,
+		path:    "/admin/banner",
```
Comment (john-m-liu) [inline rest/client/rest.go]: I think these paths need to not have the leading /, otherwise there will be 2 in the url

File: cli/admin.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+		"problem enabling services")
+
+}
+
+func setServiceFlagValues(args []string, target bool, flags *model.APIServiceFlags) error {
+	catcher := grip.NewSimpleCatcher()
+
+	for _, f := range args {
+		switch f {
+		case "dispatch", "tasks", "taskdispatch", "task_dispatch":
```
Comment (bsamek) [inline cli/admin.go]: hyphenate task-dispatch

File: rest/client/rest.go [hunk 4]
```diff
@@ -126,6 +125,77 @@ func (c *communicatorImpl) GetHosts(ctx context.Context, f func([]*model.APIHost
...
+	})
+
+	return errors.Wrap(err, "problem setting banner")
+}
+
+func (c *communicatorImpl) GetBannerMessage(ctx context.Context) (string, error) {
+	info := requestInfo{
+		method:  get,
+		version: apiVersion2,
+		path:    "admin",
```
Comment (bsamek) [inline rest/client/rest.go]: admin/banner?

File: rest/client/rest.go [hunk 5]
```diff
@@ -126,6 +125,77 @@ func (c *communicatorImpl) GetHosts(ctx context.Context, f func([]*model.APIHost
...
+	}
+
+	return nil
+}
+
+func (c *communicatorImpl) GetServiceFlags(ctx context.Context) (*model.APIServiceFlags, error) {
+	info := requestInfo{
+		method:  get,
+		version: apiVersion2,
+		path:    "admin",
```
Comment (bsamek) [inline rest/client/rest.go]: admin/service_flags?

File: rest/client/rest.go (hunk 4 above)
Comment (tychoish) [reply rest/client/rest.go]: admin/banner and admin/service_flags are PUT only, to get, use the /admin route.

**Pull request merged**